src/test/java/com/github/wovnio/wovnjava/QueryUrlLanguagePatternHandlerTest.java

```diff
@@ -47,8 +61,20 @@ public void testGetLang__NonMatchingQuery__ReturnDefaultLang() {
         assertEquals(this.defaultLang, sut.getLang("http://site.com?wovn="));
     }
 
+    public void testGetLang__NonMatchingQuery__ReturnDefaultLangByChinese() {
```

Comment:
I would say the "NonMatching" tests don't need separate test cases for zh-CHT. These tests just assert that the default language is returned so there isn't any special behavior this is testing.